Lib/test/test_argparse.py

```diff
@@ -4958,6 +4958,45 @@ def custom_type(string):
     version = ''
 
 
+class TestHelpUsageLongSubparserCommand(TestCase):
+    """Test that subparser commands are formatted correctly in help"""
+
```

Comment:
```suggestion
    maxDiff = None

```
This produces better error report if the test fails.